core/toolbox/toolbox.js

```diff
@@ -531,7 +540,7 @@ Blockly.Toolbox.prototype.getWorkspace = function() {
 
 /**
  * Gets the selected item.
- * @return {?Blockly.ToolboxItem} The selected item, or null if no item is
+ * @return {?Blockly.IToolboxItem} The selected item, or null if no item is
```

Comment:
`ISelectableToolboxItem`. Same for the next couple methods.

Reply:
Done.